python/dgl/nn/mxnet/hetero.py

```diff
@@ -8,12 +8,27 @@ class HeteroGraphConv(nn.Block):
     r"""A generic module for computing convolution on heterogeneous graphs.
 
     The heterograph convolution applies sub-modules on their associating
-    relation graphs, which reads the features from source nodes and writes the
+    relation graphs in parallel, which reads the features from source nodes and writes the
```

Comment:
"in parallel" may cause confusion since we are not actually parallelizing the computation. I think the original description is fine esp. with the new pseudo code.

Reply:
Done.